src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -24,32 +54,14 @@ public void Start(ApplicationData applicationData)
             var top = Application.Top;
 
             // Creates the top-level window to show
-            var win = new Window(applicationData.Title ?? "Out-ConsoleGridView")
+            var win = new ConsoleGuiWindow(applicationData.Title ?? "Out-ConsoleGridView")
             {
                 X = 0,
-                Y = 1, // Leave one row for the toplevel menu
+                Y = 0, 
                 // By using Dim.Fill(), it will automatically resize without manual intervention
                 Width = Dim.Fill(),
                 Height = Dim.Fill()
             };
-            top.Add(win);
-
-            // Creates a menubar, the item "New" has a help menu.
-            var menu = new MenuBar(new MenuBarItem []
-            {
-                new MenuBarItem("_Actions (F9)", 
-                    applicationData.PassThru
-                    ? new MenuItem []
-                    {
-                        new MenuItem("_Accept", "", () => { if (Quit("Accept", ACCEPT_TEXT)) Application.RequestStop(); }),
-                        new MenuItem("_Cancel", "", () =>{ if (Quit("Cancel", CANCEL_TEXT)) _cancelled = true; Application.RequestStop(); })
-                    }
-                    : new MenuItem []
-                    {
-                        new MenuItem("_Close", "", () =>{ if (Quit("Close", CLOSE_TEXT)) Application.RequestStop(); })
-                    })
-            });
-            top.Add(menu);
```

Comment:
Why was the code removed?

Reply:
I removed the menu completely because it's superfluous and adds complexity. Per Tyler, it was originally in the implementation simply because it came from a sample he copied, not because it was thought through.

Reply:
@tig Thanks for clarify! Common rule is to keep a PR as small and simple as possible and does not add unrelated changes - otherwise it complicates and slows down code reviews, 
also spoils the history of commits.

Reply:
Thanks for the coaching. I only play a software developer on TV, so my muscles on all this are pretty weak. 

FWIW, I'm curious if you are suggesting I should have pulled the menus out as one PR and introduced ESC/ENTER as a separate PR?

Reply:
> you are suggesting I should have pulled the menus out as one PR and introduced ESC/ENTER as a separate PR?

It would be best practice but I am not maintainer in the repo so I delegate this to @TylerLeonhardt .

Reply:
>  I only play a software developer on TV, so my muscles on all this are pretty weak.

Never mind. It is great project to train brain and get great skills.